.github/infrastructure/conformance/azure/conf-test-azure-eventhubs.bicep

```diff
@@ -118,6 +123,48 @@ resource eventHubsNamespace 'Microsoft.EventHub/namespaces@2017-04-01' = {
       name: certificationConsumerGroupName2
     }
   }
+  resource certificationEventHubPubsubTopicMulti1 'eventhubs' = {
+    name: certificationEventHubPubsubTopicMulti1Name
+    properties: {
+      messageRetentionInDays: 1
+    }
+    resource certificationEventHubPubsubTopicMulti1Policy 'authorizationRules' = {
+      name: certificationEventHubPubsubTopicMulti1PolicyName
+      properties: {
+        rights: [
+          'Send'
+          'Listen'
+        ]
+      }
+    }
+    resource eventHubPubsubConsumerGroup1 'consumergroups' = {
+      name: certificationConsumerGroupName1
+    }
+    resource eventHubPubsubConsumerGroup2 'consumergroups' = {
+      name: certificationConsumerGroupName2
+    }
+  }
+  resource certificationEventHubPubsubTopicMulti2 'eventhubs' = {
+    name: certificationEventHubPubsubTopicMulti2Name
+    properties: {
+      messageRetentionInDays: 1
+    }
+    resource certificationEventHubPubsubTopicMulti2Policy 'authorizationRules' = {
+      name: certificationEventHubPubsubTopicMulti2PolicyName
+      properties: {
+        rights: [
+          'Send'
+          'Listen'
+        ]
+      }
+    }
+    resource eventHubPubsubConsumerGroup1 'consumergroups' = {
+      name: certificationConsumerGroupName1
+    }
+    resource eventHubPubsubConsumerGroup2 'consumergroups' = {
+      name: certificationConsumerGroupName2
+    }
```

Comment:
Please change to a single consumer group with name `eventHubPubsubConsumerGroupName`:

```bicep
    resource eventHubPubsubConsumerGroup 'consumergroups' = {
      name: eventHubPubsubConsumerGroupName
    }
```

Reply:
Done